pkg/p2p/libp2p/internal/handshake/handshake.go

```diff
@@ -33,7 +34,7 @@ const (
 	StreamName = "handshake"
 	// MaxWelcomeMessageLength is maximum number of characters allowed in the welcome message.
 	MaxWelcomeMessageLength = 140
-	messageTimeout          = 5 * time.Second
+	handshakeTimeout        = 15 * time.Second
```

Comment:
why the increase in timeout? this is quite a lot of time no?

Reply:
This is now a different timeout. `messageTimeout` covered only one message in the handshake, while `handshakeTimeout` covers the whole handshake process. I think that this timeout should be a protective one. But it is completely set by a guess by me. Would 5s for the whole handshake be more reasonable?

Reply:
I see. Let's leave it as is for now :+1: